charts/orchestrator-k8s/locations/template.yaml

```diff
@@ -0,0 +1,180 @@
+apiVersion: scaffolder.backstage.io/v1beta3
+kind: Template
```

Comment:
I don't think this template will work without this folder: https://github.com/parodos-dev/workflow-software-templates/tree/main/scaffolder-templates/complex-assessment-workflow/skeleton

we can have only the basic one included in the chart.

Reply:
done